c2corg_ui/static/js/map/map.js

```diff
@@ -451,7 +451,7 @@ app.MapController.prototype.createPointStyle_ = function(feature, resolution) {
 
```

Comment:
Yes I know ... I need to clean the branch, but now priority is to fix, after I clean ...